includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -1946,6 +2006,10 @@ private function finalize_styles() {
 				}
 			}
 
+			if ( ! self::has_required_php_css_parser() ) {
+				$comment .= "\n" . esc_html__( '!!!WARNING!!! AMP CSS processing is limited because a conflicting version of PHP-CSS-Parser has been loaded by another plugin/theme. Tree shaking is not available.', 'amp' ) . "\n";
+			}
```

Comment:
This looks good when the *Mailpoet* plugin is active.

<img width="1316" alt="warning-appears" src="https://user-images.githubusercontent.com/4063887/49979701-c6d45380-ff15-11e8-9d87-ed05db55140b.png">